package.json

```diff
@@ -244,5 +252,10 @@
     "webpack-dev-middleware": "^5.3.1",
     "webpack-hot-middleware": "^2.25.1",
     "xhr": "^2.6.0"
+  },
+  "eslintConfig": {
```

Comment:
We have an existing `.eslintrc` file so this needs to be put there instead.
https://github.com/processing/p5.js-web-editor/blob/3493b3b8099d42cac7220f0181ba4fbafebddbf0/.eslintrc#L2

This is why you are getting errors on the "Test and lint code base" Github action.

Reply:
We could probably remove this for now and re-include it in the future. This was a suggestion by storybook when initialising

Reply:
To minimise the changes in this PR I reverted this. Can always re-include it later.

```
  "eslintConfig": {
    "extends": [
      "plugin:storybook/recommended"
    ]
  }
```